### PR TITLE
Added Ubisoft.Connect v122.1.10474

### PR DESCRIPTION
**Pull request by OfficialEsco**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15910)

### CODE REVIEW

File: manifests/u/Ubisoft/Connect/122.1.0.10474/Ubisoft.Connect.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate
+
+PackageIdentifier: Ubisoft.Connect
+PackageVersion: 122.1.0.10474
```
Comment (jedieaston) [inline manifests/u/Ubisoft/Connect/122.1.0.10474/Ubisoft.Connect.installer.yaml:4]:
```suggestion
PackageVersion: 122.1.10474
```

File: manifests/u/Ubisoft/Connect/122.1.0.10474/Ubisoft.Connect.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate
+
+PackageIdentifier: Ubisoft.Connect
+PackageVersion: 122.1.0.10474
```
Comment (jedieaston) [inline manifests/u/Ubisoft/Connect/122.1.0.10474/Ubisoft.Connect.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 122.1.10474
```

File: manifests/u/Ubisoft/Connect/122.1.0.10474/Ubisoft.Connect.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Ubisoft.Connect
+PackageVersion: 122.1.0.10474
```
Comment (jedieaston) [inline manifests/u/Ubisoft/Connect/122.1.0.10474/Ubisoft.Connect.yaml:4]:
```suggestion
PackageVersion: 122.1.10474
```

**Review by jedieaston — CHANGES_REQUESTED** [review]
To match the ARP table, can you clean these up real fast? (Also, you'll need to move the manifest to manifests/u/Ubisoft/Connect/122.1.10474/).

**Comment by OfficialEsco** [comment]
@denelon here is another issue where WinGetBot finds the "incorrect" version, however `File version` and `Product version` shows the same wrong version..
image
image

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**